lib/i18n/activities.js

```diff
@@ -80,3 +80,25 @@ export const ActivityCategoryLabelI18n = defineMessages({
     defaultMessage: 'Fund & Event',
   },
 });
+
+export const ActivityAttributionI18n = defineMessages({
+  ALL: {
+    id: 'ActivityAttribution.AllShort',
+    defaultMessage: 'All',
+  },
+  AUTHORED: {
+    id: 'ActivityAttribution.AUTHORED',
+    defaultMessage: 'Authored',
+  },
+  RECEIVED: {
+    id: 'ActivityAttribution.RECEIVED',
+    defaultMessage: 'Received',
+  },
+  SELF: {
+    id: 'ActivityAttribution.SELF',
+    defaultMessage: 'Self actions',
+  },
+  HOSTED_ACCOUNTS: {
+    defaultMessage: 'Hosted Accounts',
+  },
+});
```

Comment:
I was thinking these labels such as `Recieved`, `Self actions` etc might not be self explanatory on exactly what they filter.  Suggest maybe adding tooltips when hover over them so that the user better understands what for example "Self actions" mean. 🤔

Reply:
I'm not sure about having a tooltip since it's in a select, but I agree they're not self-explanatory. I'll request some feedback about this in a demo and will iterate accordingly.